### DISCUSSION TITLE
Added ability to select beatmap from player profile to queue in multiplayer mode.

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#23956] [Ideas] [answered]
The story is that I'm a bit of a multiplayer person and get in the habit of checking someone's profile from time to time to pick the 2 beatmaps that are their most played and the beatmap with their highest performance score. The goal could be to enjoy the taste of music and understand the level they are currently at. If it's possible to simply add to the beatmap queue by clicking on the path directly from the profile beatmap like in the video below, that would be very convenient.

https://github.com/ppy/osu/assets/128424762/a1779636-32af-4265-970e-19ca750b2622

**Comment by peppy** [comment, marked as answer]
You should be able to do this already, but the process is a bit weird because you will need to click the "go to beatmap" after opening the "add item" screen.

https://github.com/ppy/osu/assets/191335/6503a688-52a5-45c3-830c-d86dde942233

We could probably attempt to make this work from the lobby screen automatically, but it's *not impossible* to already do this with a bit of effort.

**Reply by huyenden** [reply]
This works for beatmaps that have not been downloaded as in your example video, a new completed download will show a message leading to the beatmap when at "add item" to lead to that beatmap, but with beatmaps already once loaded, it won't appear like in my example clip.

Anyhow, if peppy can optimize this, I'll be very grateful

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/23974
